stormtrooper/openai.py

```diff
@@ -139,9 +135,7 @@ def __init__(
             openai.api_key = os.environ["OPENAI_API_KEY"]
             openai.organization = os.environ.get("OPENAI_ORG")
         except KeyError as e:
-            raise KeyError(
-                "Environment variable OPENAI_API_KEY not specified."
-            ) from e
+            raise KeyError("Environment variable OPENAI_API_KEY not specified.") from e
```

Comment:
I suppose this could also be OPENAI_ORG?

Reply:
OPENAI_ORG is optional